packages/react/plugins/babel.ts

```diff
@@ -0,0 +1 @@
+module.exports = require('../src/plugins/babel').getBabelWebpackConfig;
```

Comment:
why do we have this file? can't we just use the src/plugins/babel? or inline that file here?